python/pyarrow/tests/test_pandas.py

```diff
@@ -182,6 +187,8 @@ def test_column_index_names_are_preserved(self):
         df.columns.names = ['a']
         _check_pandas_roundtrip(df, preserve_index=True)
 
+    @pytest.mark.filterwarnings(
+        "ignore:parsing timezone aware datetimes:DeprecationWarning")
```

Comment:
Do you remember for which pandas version this happened?

Reply:
I don't I was trying locally between a couple Python version environments and I too cannot get it to recreate.. so trying in CI now https://github.com/apache/arrow/pull/14729/commits/a599d65fd09579c5d1a4195a7f41f356b958d544 (same for comment about 'Discarding nonzero nanoseconds')

Reply:
Yeah, it might be that bumping the minimum pandas version to 1.0 in the meantime has made that disappear already

Reply:
Okay, well it's not in CI that I can see.. not sure how I managed to get that. will remove the filter.